pages/docs/support/rest-api.mdx

```diff
@@ -0,0 +1,215 @@
+As Grouparoo can be self-hosted, every deployment of Grouparoo provides its own API documentation.
+The Swagger docs for your API can be found by navigating to `/swagger` in the Grouparoo UI.
+
+<ImageInBrowserFrame
+  centered
+  source="Generate an API Key"
+  alt="Generate an API Key"
+  src="/images/docs/support/rest-api/swaggerdocs.png"
+  width={820}
+  height={530}
+/>
+
+The Grouparoo API can be authorized in 2 distinct ways: as a web user (with a cookie and CSRF token), or via an API Key. By default, when visiting the `/swagger` page, you will be authenticated using your existing cookie. You can switch to using an API key by clicking "Authorize" and entering your API Key (if pasting in your API Key, be careful about whitespace).
+
+<ImageInBrowserFrame
+  centered
+  source="Generate an API Key"
+  alt="Generate an API Key"
+  src="/images/docs/support/rest-api/auths.png"
+  width={820}
+  height={521}
+/>
+
+For example, if you want to see a list of all your instance's apps, you can run:
+
+```bash
+# Set Variables
+GROUPAROO_HOST="http://localhost:3000"
+GROUPAROO_API_KEY="ba05b18d9dae4acbb08799e7ef238f0d"
+
+# Make Requests
+curl -X GET "$GROUPAROO_HOST/api/v1/apps?apiKey=$GROUPAROO_API_KEY"
+```
+
+You can also make post requests. Running the command above, you should have gotten back a list of all your apps. You can use an `app_id` to generate a source like so:
+
+```bash
+# Set new variables for the request
+APP_ID="demo_db"
+SOURCE_NAME="demo_source"
+TABLE_NAME="users"
+SOURCE_TYPE="postgres-table-import"
+
+# Make Request
+curl \
+  -X POST \
+  -d "apiKey=$GROUPAROO_API_KEY" \
+  -d "appId=$APP_ID" \
+  -d "name=$SOURCE_NAME" \
+  -d "type=$SOURCE_TYPE" \
+  -d "state=ready" \
+  -d "options={\"table\": \"$TABLE_NAME\"}" \
+  -d "mapping={\"id\": \"user_id\"}" \
+"$GROUPAROO_HOST/api/v1/source"
+```
+
+You'll get back an object that includes the source's id:
+
+```json
+{
+  "source": {
+    "id": "src_a654f21f-b745-4ee3-824e-3a9d039a405f",
+    "name": "demo_source",
+    "type": "postgres-table-import",
+    "state": "ready",
+    "mapping": {},
+    "app": {
+      "id": "demo_db",
+      "name": "Demo Database",
+      "icon": "/public/@grouparoo/postgres/postgres.svg",
+      "type": "postgres",
+      "state": "ready",
+      "locked": "config:code",
+  ...
+    }
+  }
+}
+```
+
+Then, you can use the source id to create a property
+
+```bash
+# Set new variables for the request
+APP_ID="demo_db"
+PROPERTY_ID="email"
+SOURCE_ID="demo_source"
+PROPERTY_KEY="email"
+PROPERTY_TYPE="email"
+COLUMN="email"
+AGGREGATION_METHOD="exact"
+
+# Make Request
+curl \
+  -X POST \
+  -d "apiKey=$GROUPAROO_API_KEY" \
+  -d "id=$PROPERTY_ID" \
+  -d "source=$SOURCE_ID" \
+  -d "key=$PROPERTY_KEY" \
+  -d "type=$PROPERTY_TYPE" \
+  -d "state=ready" \
+  -d "options={\"column\": \"$COLUMN\", \"aggregationMethod": \"$AGGREGATION_METHOD\"}" \
+
+"$GROUPAROO_HOST/api/v1/source"
+
+# Make Request
+
+curl -X POST "http://localhost:3000/api/v1"
```

Comment:
What's this do?  The POST to `"$GROUPAROO_HOST/api/v1/source"` already happened above?

Reply:
copy/paste fail